plenum/test/node_request/test_2_nodes_got_only_preprepare.py

```diff
@@ -69,6 +69,8 @@ def test_2_nodes_get_only_preprepare(looper,
            master_node.master_last_ordered_3PC[1]
 
     # 2d behind got 1 stashed checkpoint from all nodes except 1st behind
+    check_num_received_checkpoints(behind_nodes[1].master_replica, 1)
+    check_received_checkpoint_votes(behind_nodes[1].master_replica, )
```

Comment:
Are there missing parameters?

Reply:
Yes, accidentally half-changed, already fixed it